### ISSUE TITLE
Forced subtitles with 5.13 Vidcoder

### ISSUE THREAD

**Issue by LeKeiser** [RandomEngy/VidCoder#602]
Hello,
I've just upgraded Vidcoder to version 5.13, and the subtitles always are always burned. I cannot untick them. I checked my videos with Handbrake, I can add the subtitles to the rips, and choose if I want to see them or not. I reinstalled Vidcoder v5.12 Beta and I can choose whether or not I want the subtitles to be burned or not. 
I have uninstalled and reinstalled Vidcoder 5.13, and again, I have no choice. Either I choose subtitles and it will be burned within the rip. Or I don't, and I have... no subtitles :(
Is this a bug? 
Thanx for your help :)
Cheers,
LeK

**Comment by RandomEngy** [comment]
What container are you using, and what subtitle type are you trying to do this with? There are some restrictions on when you can include a "soft" subtitle and when you have to burn.

**Comment by LeKeiser** [comment]
Hello,
Thank you for your answer.
I'm compressing a MKV file (I use MakeMKV) using H265, and the container will still be a MKV.
I haven't changed my configuration from 5.12 to 5.13. Exactly the same. But 5.13 doesn't give me the option to add the subtitles (they are already in the source MKV) without them to be burned. Either I add the subtitles and they are burned, or I don't.
I have reinstalled VidCoder 5.12 and no problem. If I use the portable version of 5.13, there we go. No problem with Handbrake.

**Comment by RandomEngy** [comment]
When you load in the source it should list something beside the subtitles, like (VobSub) or (PGS). Can you tell me what that one is?

**Comment by LeKeiser** [comment]
Hello,
Sorry for the delay. It says ¨PGS" with v5.13 and the subtitles will be burned.
If I use v5.12 with the same MKV, it's still PGS but the subtitles won't be burned, just added.

I've tried another MKV, this time I get VobSub, and the subtitles won't be burned, using v5.13.

**Comment by RandomEngy** [comment]
Just to double-check, can you tell me what you see under Encoding settings -> Container -> Container format? On 5.13, where you see the PGS subtitles as "burn only".

PGS has to be burned on MP4 but can be passed through for MKV.

**Comment by LeKeiser** [comment]
Damn!!
I was going to check the Encoding Settings when I saw that the selection was XBox Legacy 1080p chosen, and not what I always use, H.265 MKV 1080p... 
And when it's XBox, then the subtitles are burned. Not if I choose H.265 from the list.
I have uninstalled v5.12 beta and installed v5.13 and now, choosing H.265 from the list, I can add the subtitles while not being forced burned.

Really sorry , I feel I have wasted your time :(
But you put me on the right track and you've solved my "little" problem so THANX A LOT :)
